gcore/gdalpythondriverloader.cpp

```diff
@@ -773,6 +773,8 @@ OGRFeature *PythonPluginLayer::TranslateToOGRFeature(PyObject *poObj)
     PyObject *myLongType = PyObject_Type(myLong);
     PyObject *myFloat = PyFloat_FromDouble(1.0);
     PyObject *myFloatType = PyObject_Type(myFloat);
+    PyObject *myStr = PyUnicode_FromString("");
+    PyObject *myStrType = PyObject_Type(myStr);
```

Comment:
both should be Py_DecRef()'ed at the end of the method

Reply:
Good spot. I'd have much preferred to use:
- `PyUnicode_Check()` & friends, but they didn't load. Not sure if that's because they're not in the Stable ABI, or maybe related to my comment not using true libpython?
- `PyObject_IsInstance(o, PyUnicode_Type)` but that requires getting the `PyTypeObject` opaque struct to be defined, which was a bit beyond my Wednesday afternoon 😁

Reply:
Ah. At least some of the `Py*_Check()` functions are macos. That'd explain it :-)